leaflet-treering.js

```diff
@@ -25,7 +25,11 @@ function LTreering (viewer, basePath, options) {
     'savePermission': options.savePermission || false,
     'popoutUrl': options.popoutUrl || null,
     'assetName': options.assetName || 'N/A',
-    'hasLatewood': options.hasLatewood,
+  }
+
+  this.preferences = {
+    'forwardDirection': options.initialData.forwardDirection,
```

Comment:
This is going to fail if initialData is null, or if forwardDirection doesn't exist

Reply:
I account for that in MeasurementOptions.prototype.prefencesInfo() (line 2834). Does it need to be accounted for up here too?

Reply:
Sorry, resolved by accident.

You haven't called that method by the time this code is run. At the time this code is invoked ,only lines 18-29 will have executed.

Reply:
Would there be a case when initialData is null or undefined? That scenario also effects MeasurementData. Also, it works fine if forwardDirection or subAnnual don't exist (i.e. 'initialData': {} in .html)

Reply:
yeah I suppose it doesn't fail, it just results in those being undefined rather than having a default state. I guess that's ok - in general, it's nice for code to be more clear, so minimally options.initialData.forwardDiration || false